app/main.py

```diff
@@ -1 +1,109 @@
-# write your code here
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```

Comment:
The `get_rating` method in the `Player` class is defined to return `None`, but it should return an `int` as per the implementations in subclasses. Consider changing the return type annotation to `int`.